providers/aws/services/cloudfront/cloudfront_distributions_using_deprecated_ssl_protocols/cloudfront_distributions_using_deprecated_ssl_protocols.py

```diff
@@ -0,0 +1,29 @@
+from lib.check.models import Check, Check_Report
+from providers.aws.services.cloudfront.cloudfront_client import cloudfront_client
+
+SSLV3 = "SSLv3"
```

Comment:
I would put all the ones that accept TLSv1.1 or lower.
More info: https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/secure-connections-supported-viewer-protocols-ciphers.html

Reply:
SSLv3 | TLSv1 | TLSv1_2016 | TLSv1.1_2016
-- | -- | -- | --

Reply:
Good point! I've updated it, please check it out.

Reply:
I've changed the logic a little bit